src/components/darkmode-switch/darkmode-switch.tsx

```diff
@@ -0,0 +1,132 @@
+import { Component, h, State } from '@stencil/core';
+
+const DEFAULT: Theme = 'system-default';
+const LIGHT: Theme = 'force-light';
+const DARK: Theme = 'force-dark';
+const CHECKBOX_LIGHT = false;
+const CHECKBOX_DARK = true;
+const THEME_EVENT_NAME = 'kompendium-theme';
```

Comment:
I think event names are usually camel cased :thinking: 
Also, if we put this constant in a separate file we can export it and reuse it in `playground.tsx`

Reply:
so you mean for example, to be able to only switch the code-preview part of the examples to dark-mode?

Reply:
No, just so we can reuse the constant and don't have to duplicate the string for the event name in the code :slightly_smiling_face:

Reply:
⚡